services/tenant-ui/frontend/src/assets/layout.scss

```diff
@@ -1,4 +1,4 @@
-@import 'variables.scss';
+@use 'variables' as *; // Import variables as global
```

Comment:
Should this be `variables.scss` like in other locations?

Reply:
Oh, yeah doesn't matter in Sass but should be consistent everywhere. Fixed